packages/vitest/src/types/global.ts

```diff
@@ -34,6 +34,11 @@ interface InlineSnapshotMatcher<T> {
 }
 
 declare module '@vitest/expect' {
+  // Allow unused `T` to preserve its name for extensions.
+  // Type parameter names must be identical when extending those types.
+  // eslint-disable-next-line
+  interface Matchers<T> {}
```

Comment:
The changes in this file seem excessive, we already expose `Matchers` in `@vitest/expect`, we just need to also extend `Assertion` there. Is there a reason why you extend `ExpectStatic`, but don't extend `Assertion` in the `expect` package?

Reply:
See

https://github.com/vitest-dev/vitest/blob/d5765f71b3d269cb2d3a06666a5070372e21e62e/packages/expect/src/types.ts#L631

Reply:
Without this, it seems like you can't use `Matchers` type only with the `@vitest/expect` package - the `vitest` is required

Reply:
This is a great point. I put `Matchers` in `vitest` because I wasn't well familiar with the packages relationship back then. I followed your advice and moved the extension to `@vitest/expect` so that now these changes extend `ExpectStatic` and `Assertion` from `@vitest/expect` and do not touch the `vitest` changes at all. Type tests confirm that is enough. Awesome!